src/quo/components/settings/settings_item/style.cljs

```diff
@@ -54,6 +55,14 @@
             colors/white-opa-70
             (colors/theme-colors colors/neutral-50 colors/neutral-40 theme))})
 
+(defn image-color
+  [blur? image-right? theme]
+  {:color (if blur?
```

Comment:
maybe `(cond` is better in this case?

```
(cond
   (not blur?) (color blur? theme)
    image-right? colors/white-opa-40
    :else colors/white-opa-70))
```
Looks a bit more readable and it should be equivalent

Reply:
Oh yeah, thanks for pointing this out 🙏 
I think should be able to use that `cond` expression

Reply:
After looking at the code a bit more, I think I'm able to flatten the `cond` expression into:
```
(if (and blur? image-right?)
    colors/white-opa-40
    (color blur? theme))
```
That `color` function already takes care of using the `colors/white-opa-70` in the right places I think.